sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3626,6 +3626,125 @@ object functions {
    */
   def map_values(e: Column): Column = withExpr { MapValues(e.expr) }
 
+  //////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Do we need to include these expressions here?

Reply:
@gatorsmile @ueshin what do you think?

Reply:
Let's keep these for now and revisit again later. We'll need to discuss which one is here or not in a batch.